doc/man7/EVP_SIGNATURE-ML-DSA.pod

```diff
@@ -85,7 +85,7 @@ To sign a message using an ML-DSA EVP_PKEY structure:
         size_t sig_len;
         unsigned char *sig = NULL;
         const OSSL_PARAM params[] = {
-            OSSL_PARAM_octet_string("context-string", (unsigned char *)"A context string", 33),
+            OSSL_PARAM_octet_string("context-string", (unsigned char *)"A context string", 17),
```

Comment:
Should this include the \0? As far as I know, this is not about C-strings, so 16 could also be correct.

Reply:
Either is "correct", the choice of context string payload is protocol-specific.